packages/block-library/src/video/save.js

```diff
@@ -37,6 +41,8 @@ export default function save( { attributes } ) {
 					preload={ preload !== 'metadata' ? preload : undefined }
 					src={ src }
 					playsInline={ playsInline }
+					className={ dimensionsProps.className }
+					style={ dimensionsProps.style }
```

Comment:
As long as users don't add `aspectRatio` to block instances, the markup should remain the same, so I believe the block deprecation is not necessary.

Reply:
I'm not sure I'm following this comment, sorry, @t-hamano.

Isn't this code change explicitly about applying a user-selected aspect ratio on a block instance?